configure.ac

```diff
@@ -82,7 +81,6 @@ GIO_CFLAGS="$GIO_CFLAGS -DGLIB_VERSION_MAX_ALLOWED=$GLIB_VERSION_DEF"
 
 PKG_CHECK_MODULES(LIBSYSTEMD, [$LIBSYSTEMD_REQUIREMENT])
 PKG_CHECK_MODULES(JSON_GLIB, [$JSON_GLIB_REQUIREMENT])
-PKG_CHECK_MODULES(POLKIT, [$POLKIT_REQUIREMENT])
```

Comment:
Please keep this and `POLKIT_REQUIREMENT` as they are -- instead, just add an "action_failed" argument, and create an `AC_DEFINE` if polkit is available. See `LIBSSH_SAFE_PROXY_COMMANDS` for an example. That will provide proper auto-detection.

Reply:
Very good idea. Thank you.